package.json

```diff
@@ -40,21 +46,30 @@
     "grunt-wiredep": "3.0.0",
     "html-minifier": "1.1.1",
     "imagemin": "1.0.5",
+
+    "jasmine-core": "^2.8.0",
     "jasmine-beforeall": "0.1.1",
     "jasmine-spec-reporter": "1.1.2",
+
     "jshint-stylish": "0.2.0",
-    "karma": "0.12.23",
-    "karma-chrome-launcher": "^2.0.0",
-    "karma-coverage": "0.2.6",
-    "karma-firefox-launcher": "^1.0.0",
-    "karma-jasmine": "0.1.5",
-    "karma-ng-html2js-preprocessor": "1.0.0",
+
+    "karma": "^1.7.1",
+    "karma-chrome-launcher": "^2.2.0",
+    "karma-coverage": "^1.1.1",
+    "karma-firefox-launcher": "^1.0.1",
+    "karma-jasmine": "^1.1.0",
+    "karma-ng-html2js-preprocessor": "^1.0.0",
+    "karma-jasmine-diff-reporter": "^1.1.0",
     "karma-phantomjs-launcher": "^1.0.4",
+    "karma-nightmare": "^0.4.10",
+
     "less": "2.6.1",
     "load-grunt-tasks": "0.4.0",
     "lodash": "3.10.1",
-    "protractor": "1.7.0",
-    "protractor-screenshot-reporter": "0.0.5",
+
+    "protractor": "^5.1.2",
```

Comment:
Did you mean to bump protractor in this pull? I thought other changes were needed for that.

Reply:
Ah, crap, my last swap between branches must have accidentally committed that, thanks.  Will update and see where that gets us.